docs/src/man/missing.md

```diff
@@ -69,32 +71,34 @@ The function `Missings.T` returns the element-type `T` in `Union{T, Missing}`.
 
 ```jldoctest missings
 julia> eltype(x)
-Union{Int64, Missings.Missing}
+Union{Int64, Missing}

```

Comment:
And here the same question against `Base.nonmissingtype`?

Reply:
`Base.nonmissingtype` isn't part of the public API, and initially it has only been added for internal use. Hopefully we'll be able to support something like `typesubtract(T, Missing)` at some point. For now I prefer recommending `Missings.T`, which is public even if it's not exported.

Reply:
OK